assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -101,6 +73,15 @@ export default function WidgetAreaRenderer( { slug, totalAreas } ) {
 		/>
 	) );
 
+	// Here we render the bare output as it is guaranteed to render empty.
+	// This is important compared to returning `null` so that the area
+	// can maybe render later if conditions change for widgets to become active.
+	// Returning `null` here however would have the side-effect of making
+	// all widgets active again, which is why we must return the "null" output.
+	if ( ! activeWidgets.length ) {
+		return widgetsOutput;
+	}
```

Comment:
Related to my above comment: This is the only place where we should be using `activeWidgets` in this component; since we don't want to render any surrounding UI in that case.

Reply:
Updated: This is the only place for activeWidgets.